api/src/main/java/org/pmiops/workbench/billing/BillingProjectBufferService.java

```diff
@@ -51,11 +54,36 @@ public void bufferBillingProject() {
     billingProjectBufferEntryDao.save(entry);
   }
 
+  public void syncBillingProjectStatus() {
+    BillingProjectBufferEntry entry = billingProjectBufferEntryDao
+        .findFirstByStatusOrderByCreationTimeAsc(StorageEnums.billingProjectBufferStatusToStorage(CREATING));
+
+    if (entry == null) {
+      return;
+    }
+
+    switch (fireCloudService.getBillingProjectStatus(entry.getFireCloudProjectName()).getCreationStatus()) {
+      case READY:
+        entry.setStatusEnum(AVAILABLE);
+        break;
+      case ERROR:
+        entry.setStatusEnum(ERROR);
```

Comment:
It is probably worth logging an error here, and/or an error message if we have one from FC.

Reply:
I don't believe there's an error message in this code path since this is technically a 200 from FC. If the actual get request fails, we should see the stack trace getting logged.

Reply:
I did add a log for when this code path gets hit though